base/shell/explorer/explorer.cpp

```diff
@@ -87,6 +87,56 @@ HideMinimizedWindows(IN BOOL bHide)
 }
 #endif
 
+static BOOL
+IsExplorerSystemShell()
+{
+    HKEY hKeyWinlogon;
+    WCHAR szPath[MAX_PATH];
+    LPWSTR szExplorer;
+    BOOL bIsSystemShell = TRUE; // Assume we are the system shell by default.
+
+    if (!GetModuleFileNameW(NULL, szPath, _countof(szPath)))
+        return FALSE;
+
+    szExplorer = PathFindFileNameW(szPath);
+
+    if (RegOpenKeyExW(HKEY_LOCAL_MACHINE, L"Software\\Microsoft\\Windows NT\\CurrentVersion\\Winlogon",
+        0, KEY_READ, &hKeyWinlogon) != ERROR_SUCCESS)
```

Comment:
```suggestion
                      0, KEY_READ, &hKeyWinlogon) != ERROR_SUCCESS)
```
(makes newline parameters aligned)